RadixWallet/Core/Resources/Resources/en.lproj/Localizable.strings

```diff
@@ -963,4 +965,4 @@ You will be asked to enter your main seed phrase. This is a set of **24 words**
 "recoverWalletWithoutProfile_complete_headerSubtitle" = "Accounts discovered in the scan have been added to your wallet.
 
 If you have any “Legacy” Accounts (created on the Olympia network) to import - or any Accounts using a Ledger hardware wallet device - please continue and then use the **Account Recovery Scan** option in your Radix Wallet settings under **Account Security**.";
-"recoverWalletWithoutProfile_complete_continueButton" = "Continue";
```

Comment:
These temporary strings should have a bunch of newlines above them to make sure they are not accidentally added to main. Ideally even a comment line above the first one.